### PR TITLE
10.0 access_apps

### PR DESCRIPTION
**Pull request by iledarn**

### CODE REVIEW

File: access_apps/README.rst [hunk 1]
```diff
@@ -1,6 +1,14 @@
...
+There are two use cases of this module:
 
-Tested on 10.0 1be57f2825af4f3ade20a658c6f97f6cf93cc866 
+* In Odoo all members of security group ``Administration: Settings`` (``base.group_system``) have rights to install additional modules.
+  But what if you want to restrict such access even for them?
+
+* You have a user you want to permit application installation to. But you don't want him to have all rights that members of the ``Administration: Settings`` group have.
+
+There is even third specific case -  when you want to restrict application installation but leave the ability to install apps from ``Configuration >> Settings``
+(as you know some addons inherit `res.config.settings` allowing to add features by means of installation additional modules)
```
Comment (yelizariev) [inline access_apps/README.rst]: Add how it works in short (which groups are introduced)

**Comment by iledarn** [comment]
@yelizariev , done

File: access_apps/models/res_config.py [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+class ResConfigSettings(models.TransientModel):
+    _inherit = 'res.config.settings'
+
+    @api.model
+    def _install_modules(self, modules):
+        if self.env['res.users'].has_group('access_apps.group_allow_apps_only_from_settings'):
+            result = super(ResConfigSettings, self.sudo())._install_modules(modules)
+        else:
+            result = super(ResConfigSettings, self)._install_modules(modules)
```
Comment (yelizariev) [inline access_apps/models/res_config.py]: You can do the same without copy-pasting code
```
_self = self
if ...:
     _self = _self.sudo()
super(ResConfigSettings, _self)._install_modules(modules)
```

**Comment by iledarn** [comment]
@yelizariev , done

**Comment by yelizariev** [comment]
Fix lints

**Comment by iledarn** [comment]
superseded by https://github.com/it-projects-llc/access-addons/pull/66

**Pull request closed without merging**